### ISSUE TITLE
Make Styling accessible

### ISSUE THREAD

**Issue by NadjaKanunnikow** [PeetThesing/capstone-project_snippet-manager#20]
## Value proposition
As an visually impaired web developer
I want to read and use everything
In order to use the app.
## Description
Accessibility is an important task, make sure the app is readable for visually impaired and blind users.
## Acceptance criteria
- [ ] the contrast should be accessible and fit the [WCAG 2.1 ](https://www.w3.org/TR/WCAG21/#relativeluminancedef)criteria
## Tasks
- [ ] create a new branch
- [ ] create global color variables which have a contrast ratio of at least 4.5:1
- [ ] correct styling where it is necessary

**Comment by NikRoe** [comment]
LGTM
